packages/react-params/src/valueToText.tsx

```diff
@@ -39,6 +39,10 @@ export default function valueToText (type: string, value: any, swallowError = tr
     return div({}, '<unknown>');
   }
 
+  if (type === 'Balance') {
+    return toFormattedBalance({ balance: new BN(value.toString()) });
```

Comment:
I think there is a method for this already on balance type: ~`value.toBn()`

Reply:
@holygits  Ok, I will give a try if you mean converting `string | BN` to `BN`.